packages/data-stores/src/launch/selectors.ts

```diff
@@ -28,8 +28,8 @@ export const getSelectedDomain = ( state: State ): DomainSuggestions.DomainSugge
 	state.domain;
 export const getSelectedPlanProductId = ( state: State ): number | undefined => state.planProductId;
 
-export const getSelectedPlan = ( state: State ): Plan | undefined =>
-	select( PLANS_STORE ).getPlanByProductId( state.planProductId );
+export const getSelectedPlan = ( state: State, locale: string ): Plan | undefined =>
```

Comment:
I don't think we need `getSelectedPlan` anymore. After the change suggested in https://github.com/Automattic/wp-calypso/pull/49559#issuecomment-771445559 we can clean this up.